models/public/mozilla-deepspeech-0.8.2/pbmm_to_pb.py

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env python3
+#
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import argparse
+
+try:
+    from tensorflow import GraphDef
+except ImportError:
+    from tensorflow.compat.v1 import GraphDef
```

Comment:
This is unnecessary, you only need the 2nd import. The minimal TensorFlow version is 1.15, which has `compat.v1`.